lib/seed/blood_pressure_seeder.rb

```diff
@@ -94,7 +96,24 @@ def call
           updated_at: recorded_at,
           timezone_offset: 0
         }
+
+        prescription_drugs << {
+          id: SecureRandom.uuid,
+          name: "Amlodipine",
```

Comment:
Looks like we need to extract the common drug names / dosages from the protocol factory into a seed object somewhere, and then you could grab a random one here easily.  

something like...
```
drug = Seed::ProtocolDrugs.instance.sample
name = drug.name
dosage = drug.dosage
xnorm_code = drug.code
```